### DISCUSSION TITLE
能给快捷键拓展的切换倍速改成默认三倍吗，每次打开新视频都需要手动调成三倍才能使用这个切换键

### DISCUSSION THREAD

**Discussion by Godwin9527** [the1812/Bilibili-Evolved#4443] [功能建议 / Ideas]
1

**Comment by Geeyun-JY3** [comment]
[`记忆倍速`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E8%AE%B0%E5%BF%86%E5%80%8D%E9%80%9F)失效了吗？

**Reply by Godwin9527** [reply]
我不太清楚，我第一次装这个切换倍速的拓展，你的意思是我在A视频设置三倍速后可以用快捷键切换，然后我打开B视频再次按下切换倍速的快捷键会自动切换到我在A视频设置的倍速吗

**Reply by Geeyun-JY3** [reply]
你到底有没有装[`记忆倍速`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E8%AE%B0%E5%BF%86%E5%80%8D%E9%80%9F)这个组件？

**Reply by Godwin9527** [reply]
装了，但是这个我打开新视频如果之前调成了三倍速，那这个新视频就是三倍速，按切换的快捷键也能恢复到一倍，但是如果我是一倍速的时候打开新视频，那我切换就没法切换回三倍速了，只能手动提高倍速

**Reply by Geeyun-JY3** [reply]
之前好像有点搞错你的意思了。
你是否需要每个视频打开都是三倍速呢？

- 如需要，那用[`记忆倍速`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E8%AE%B0%E5%BF%86%E5%80%8D%E9%80%9F)这个组件就能解决：将`全局记忆倍速值`设为`3`，并开启`固定全局倍速值`即可，此时`快捷键扩展`中的`切换倍速`快捷键也就是在三倍速和一倍速之间切换。
- 如不需要，那就是说你只是需要能直接切换到指定倍速的快捷键，那目前应该无法满足你的需求。
  据我实测快捷键扩展中的`切换倍速`快捷键是需要配合[`记忆倍速`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E8%AE%B0%E5%BF%86%E5%80%8D%E9%80%9F)这个组件使用的。
  刚好昨天也有人提了这个需求：#4446

**Reply by Godwin9527** [reply]
对对，我就是需要 #4446 这个一键切换的功能
